com.civicrm.hrjobroles/js/src/job-roles/controllers/hr-job-roles-controller.js

```diff
@@ -913,8 +913,9 @@ define([
 
                     data.values.forEach(function (object_data) {
                         var end_date = isDateEmpty(object_data.end_date) ? null : object_data.end_date;
```

Comment:
As we're using `.startOf('day')` for `todaysDate`, shouldn't we use it for the `end_date` as well ?